test/test_google_cloud_manage.py

```diff
@@ -690,37 +725,69 @@ def test_get_service_account_from_group_mult_accounts(test_cloud_manager):
     """
     # Setup #
     test_domain = "test-domain.net"
-    primary_service_account = "primary-account" + test_domain
-
     new_member_1_id = "1"
-    new_member_1_email = new_member_1_id + "@" + test_domain
-    new_member_2_id = "2"
-    new_member_2_email = "2@" + test_domain
-
-    group_id = new_member_2_id + "-testuser"  # This has to be the user's ID - Username
+    new_member_1_username = "testuser"
```

Comment:
This seems to be duplicating some things from the previous test, maybe break some of this into fixtures?